scripts/install_build_tools.sh

```diff
@@ -4,24 +4,35 @@ set -e
 . /etc/lsb-release
```

Comment:
Is this line necessary? What is it supposed to do?

Reply:
No not really.